app.js

```diff
@@ -0,0 +1,109 @@
+'use strict';
+
+const express = require('express');
+const app = express();
+const path = require('path');
+const logger = require('./lib/logger');
```

Comment:
@JoeChapman Any idea of when the HOF-Logger will be ready? Should the old lib/logger be re-inserted until the hof-logger is ready?